pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -832,4 +849,17 @@ public void testSqlDistinctQueryCompilation() {
     Assert.assertEquals(aggregationInfo.getAggregationParams().get(FunctionCallAstNode.COLUMN_KEY_IN_AGGREGATION_INFO),
         "add(div(col1,col2),mul(col3,col4)):sub(col3,col4):col5:col6");
   }
+
+  @Test
+  public void testQueryValidationFailure() {
```

Comment:
Add one more test to cover cases like "select A, count(*) from table group by A,B"? It is supposed to another validation error.

Reply:
I thought this is a valid query ?

Reply:
added this to the test